### ISSUE TITLE
importing data from one database to the new database

### ISSUE THREAD

**Issue by jbmcclain** [YetiForceCompany/YetiForceCRM#4369]
#### Issue
I have an old version of yetiforice 2.3.1491 I have been able to export everything out and back in except for one table.

the table i am trying to import needs to keep the user information that of who entered it in and the date and time it was entered in.

I am trying to bring this in version 4.0.0 

browser firefox chrome most current release same affect.
xampp
on 2008 R@

**Comment by skramanzia** [comment]
I don't know if that is possible, you can try to update one by one the versions until reaching the current.

https://github.com/YetiForceCompany/UpdatePackages

**Comment by jbmcclain** [comment]
after modifying the version number slightly I was able to completely update.  Is there a way to be sure that the whole system has been updated correctly? I noticed in the 2.3.0 to 3.0.0 it was hard to follow.  is there some type of script that runs to verify everything is in order?
